src/Statistics.ts

```diff
@@ -0,0 +1,4 @@
+export class Statistics {
+    public dataScannedInBytes: number | undefined;
+    public executionTimeInSeconds: number | undefined;
+}
```

Comment:
Como es algo que nunca instancias ni tiene métodos ni nada, puedes pasarla a interfaz.

```suggestion
export interface Statistics {
   readonly dataScannedInBytes?: number;
   readonly executionTimeInSeconds?: number;
}
```